src/App.js

```diff
@@ -25,22 +26,12 @@ const alex = {
 
 const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">My wife&apos;s name is Natasha</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">My husband&apos;s name is Maksym</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+
+    <Person {...misha} />
+
+    <Person {...olya} />
+
+    <Person {...alex} />
```

Comment:
Don't add empty lines between single-line siblings.
It's useful only for multiline elements.
```suggestion
    <Person {...misha} />
    <Person {...olya} />
    <Person {...alex} />
```